pkgs/development/tools/cdecrypt/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, stdenv, fetchFromGitHub }:
+stdenv.mkDerivation rec {
```

Better readability and cleaner diffs in the future if one entry per line.

```suggestion
{ lib
, stdenv
, fetchFromGitHub
}:

stdenv.mkDerivation rec {
```

---

change applied in rebased commit.